docs/pages/api-docs/data-grid/data-grid.md

```diff
@@ -163,13 +163,13 @@ Api of the `components` props of type `GridSlotsComponent`
 |  | <span class="prop-name">.MuiDataGrid-window</span> | Styles applied to the window element.|
 |  | <span class="prop-name">.MuiDataGrid-viewport</span> | Styles applied to the viewport element.|
 | <span class="prop-name">row</span> | <span class="prop-name">.MuiDataGrid-row</span> | Styles applied to the row element.|
-|  | <span class="prop-name">.Mui-selected</span> | Styles applied to the selected row element.|
-|  | <span class="prop-name">.MuiDataGrid-cellWithRenderer</span> | Styles applied to the customized cell element.|
+|  | <span class="prop-name">.Mui-selected</span> | Pseudo-class applied to the selected elements.|
```

Comment:
```suggestion
```

Reply:
Not sure why do we need to remove that one?

Reply:
Since the core documents them. you can ignore my comment.

![image](https://user-images.githubusercontent.com/42154031/122047210-3dd45680-cdb6-11eb-8015-af2dc34a181e.png)